actions/code-release/action.yml

```diff
@@ -54,13 +54,15 @@ runs:
           AWS_SECRET_ACCESS_KEY: ${{inputs.secret-key}}
           SOURCE_DIR: './${{inputs.working-folder-name}}'
           DEST_DIR: '${{inputs.s3-layer-folder}}'
-      - name: Lambda code deploy
-        uses: appleboy/lambda-action@master
+      - name: Configure AWS Credentials  
+        uses: aws-actions/configure-aws-credentials@v1
         with:
-            aws_access_key_id: ${{inputs.access-key}}
-            aws_secret_access_key: ${{inputs.secret-key}}
-            aws_region: eu-west-2
-            function_name: ${{inputs.lambda-name}}
-            zip_file: ${{inputs.working-folder-name}}/${{inputs.zip-file-name}}
-            s3_key: ${{inputs.zip-file-name}}
-
+            aws-access-key-id: ${{inputs.access-key-staging}}
+            aws-secret-access-key: ${{inputs.secret-key-staging}}
+            aws-region: eu-west-2
+      - name: Lambda code deploy on AWS
+        shell: bash
+        run: |
```

Comment:
Why didn't you use appleboy at the end?

Reply:
Appleboy Githib Action does not allow to update the tags in the lambda, which we need in order for the terraform to detect that the lambda was already updated. Without updating the tags, teraform would want to update the code for second time. So we have to use AWS CLI. To simplify the code, I also changed the code update to a command replacing two steps with one.